api/v1alpha1/oadp_types.go

```diff
@@ -34,22 +34,16 @@ const ReconcileCompleteMessage = "Reconcile complete"
 
 const OadpOperatorLabel = "openshift.io/oadp"
 const RegistryDeploymentLabel = "openshift.io/oadp-registry"
-const DataMoverDeploymentLabel = "openshift.io/oadp-data-mover"
```

Comment:
We may still need this if we have code to delete the 1.2 datamover deployment on upgrade to 1.3

Reply:
labels volume-snapshot-mover deployment leftover has
- app.kubernetes.io/component: volume-snapshot-mover
- app.kubernetes.io/instance: volume-snapshot-mover
- app.kubernetes.io/managed-by: oadp-operator
- app.kubernetes.io/name: oadp-operator-velero
- openshift.io/oadp: 'True'
- openshift.io/oadp-data-mover: 'True'

Best one to use is `openshift.io/oadp-data-mover`?